Channel names that are parsed as ints from yaml cause panic

```
panic: interface conversion: interface {} is int, not string

goroutine 1 [running]:
main.recurseChannelMap(0x779340?, 0xc00040fe90?, {0x74a400?, 0xc000010558?})
	/app/mumble_createchannels.go:184 +0x2e7
main.main.func1(0xc000020780)
	/app/mumble_createchannels.go:130 +0xa9f
github.com/urfave/cli/v2.(*App).RunContext(0xc00012e1c0, {0x856d50?, 0xc00001c080}, {0xc000014070, 0x7, 0x7})
	/go/pkg/mod/github.com/urfave/cli/v2@v2.17.1/app.go:395 +0xfe8
github.com/urfave/cli/v2.(*App).Run(...)
	/go/pkg/mod/github.com/urfave/cli/v2@v2.17.1/app.go:252
main.main()
	/app/mumble_createchannels.go:141 +0x38f
````

Workaround is to explicitly quote any numeric channel names

--- a/.bumpversion.cfg
+++ b/.bumpversion.cfg
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.1.0
+current_version = 0.1.1
 commit = False
 tag = False
 

--- a/mumble_createchannels.go
+++ b/mumble_createchannels.go
@@ -19,7 +19,7 @@ import (
 var Rev = ""
 
 // Version is incremented using bump2version
-const Version = "0.1.0"
+const Version = "0.1.1"
 
 func main() {
 	app := &cli.App{

--- a/mumble_createchannels.go
+++ b/mumble_createchannels.go
@@ -194,7 +194,14 @@ func recurseChannelMap(client *gumble.Client, parent *gumble.Channel, children i
 			log.Error("'name' not found in item #", idx)
 			continue
 		}
-		childch := addAndCheck(client, parent, name.(string))
+		childname := ""
+		switch name.(type) {
+		case string:
+			childname = name.(string)
+		case int:
+			childname = strconv.Itoa(name.(int))
+		}
+		childch := addAndCheck(client, parent, childname)
 		if childch == nil {
 			log.Error("Could not create child ", name)
 			continue

